tox.ini

```diff
@@ -4,6 +4,6 @@ envlist=py27,py26,py33,py34,py35
 
 [testenv]
 commands=
-  pytest --junitxml={envlogdir}/junit-{envname}.xml []
+  pytest --junitxml={envlogdir}/junit-{envname}.xml testing []
```

Comment:
This is incorrect as it would always enforce all tests, the testdirs ini option should be used